README.md

```diff
@@ -25,6 +25,8 @@ $ npm install
 $ npm run build
 ```
 
+- For building project on Windows we recommend to use [Cygwin](https://www.cygwin.com/) with rsync package.
```

Comment:
let's also remove `(currently available for Linux and Mac only)` from line 22, if you were able to successfully build extension on Windows

Reply:
Sounds good